third_party/Makefile

```diff
@@ -29,3 +30,10 @@ bandwidthTest: sb_micro_path
 	git clone -b v$(shell nvcc --version | grep 'release' | awk '{print $$6}' | cut -c2- | cut -d '.' -f1-2) https://github.com/NVIDIA/cuda-samples.git ./cuda-samples
 	cd ./cuda-samples/Samples/bandwidthTest && make clean && make TARGET_ARCH=x86_64 SMS="70 75 80 86"
 	cp -v ./cuda-samples/Samples/bandwidthTest/bandwidthTest $(SB_MICRO_PATH)/bin/
+# Build nccl-tests.
+# The version we use is commit 44df0bf from master branch, due to it didn't update release tag for long time.
+nccl_tests: sb_micro_path
+ifneq (,$(wildcard nccl-tests/Makefile))
+	cd nccl-tests && make MPI=1 MPI_HOME=/usr/local/mpi/ -j
```

Comment:
can we confirm the mpi location is always `/usr/local/mpi/`?

Reply:
change to use whereis to find mpi install path

Reply:
Yifan suggests to make it a variable

Reply:
OK